models/balancer/arbitrum/balancer_v2_arbitrum_pools_tokens_weights.sql

```diff
@@ -0,0 +1,62 @@
+{{
+    config(
+        schema="balancer_v2_arbitrum",
```

Comment:
minor nit: can we use single quotes on schema, like other properties? this would apply to all models

Reply:
Sure thing, this is now fixed.